microsoft/PowerToys
# Powertoys Run | VSCodeWorkspaces- add support for vscode 1.64 - bug fix #15247

## Summary of the Pull Request

**What is this about:**
VSCode insiders 1.64 has a new file for workspaces. This is pull request is to add support to the new file.

**What is included in the PR:** 
Use the new file Backups/workspaces.json instead of storage.json

**How does someone test / validate:** 
- Install lastest version of vscode-insiders.
- Open multiple project folders/workspaces or remote containers
- by typing { in Powertoys all the projects and workspaces from vscodeinsiders should appear in the result list

## Quality Checklist

- [x] **Linked issue:** #15247

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
